File: JDI/src/main/java/pages/LoginPage.java
package pages;

import org.openqa.selenium.By;

import static com.codeborne.selenide.Selenide.$;

public class LoginPage {
    private static final By USERNAME_INPUT = By.id("id_username");
    private static final By PASSWORD_INPUT = By.id("id_password");
    private static final By LOGIN_BUTTON = By.id("main_action_form_button");

    public void login(String username, String password) {
        $(USERNAME_INPUT).val(username);
        $(PASSWORD_INPUT).val(password);
        $(LOGIN_BUTTON).click();
    }

}

File: C#/PageObjects/Pages/LoginPage.cs
using OpenQA.Selenium;

namespace PageObjects.Pages
{
    public class LoginPage
    {
        private readonly IWebDriver _driver;

        public LoginPage(IWebDriver driver)
        {
            _driver = driver;
        }

        private IWebElement UserNameInput
        {
            get { return _driver.FindElement(By.Id("id_username")); }
        }

        private IWebElement PasswordInput
        {
            get { return _driver.FindElement(By.Id("id_password")); }
        }

        private IWebElement LoginButton
        {
            get { return _driver.FindElement(By.Id("main_action_form_button")); }
        }

        public void Login(string user, string pass)
        {
            UserNameInput.SendKeys(user);
            PasswordInput.SendKeys(pass);
            LoginButton.Click();
        }
    }
}
